### PR TITLE
Added select all checkboxes for evaluators

### PR DESCRIPTION
**Pull request by alexvlsem**
[US134409](https://rally1.rallydev.com/#/?detail=/userstory/619024803845&fdp=true): FACE > Evaluators Assignments Table - Add select all checkboxes for evaluators

### CODE REVIEW

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js [hunk 1]
```diff
@@ -71,6 +72,11 @@ class ActivityAdvancedEvaluatonSettings extends ActivityEditorMixin(RtlMixin(Loc
 	}
 
 	async cancelChanges() {
+
+		if (!this._state || !this._state.isEnabled) {
+			return html``;
```
Comment (MingoLi) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js:77]: Do we need to return empty HTML here? or only a return is enough?
Same for the `save()` below

**Review by jmurray-d2l — APPROVED** [review]

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js [hunk 2]
```diff
@@ -238,6 +249,23 @@ class ActivityAdvancedEvaluatonSettings extends ActivityEditorMixin(RtlMixin(Loc
 		`;
 	}
 
+	_renderEvaluatorCheckbox(evaluator)
+	{
+		const selectedLearners = evaluator.selectedLearners + this._state.countToggled(evaluator.href);
```
Comment (MingoLi) [inline components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js:254]: does this calculation consider this case when you have a learner selected, then the toggle actually deselects it?

Comment (alexvlsem) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js:254]: Yes it does in 	
countToggled(evaluatorHref) {

		let count = 0;
		this._toggled.forEach((evalSet, learnerHref) => {

			if (evalSet.has(evaluatorHref)) {
				count += this.isSelected(learnerHref, evaluatorHref) ? -1 : 1;
			}
		});

		return count;
	}

**Review by gfrancu — DISMISSED** [review]

File: components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js (hunk 1 above)
Comment (alexvlsem) [reply components/d2l-activity-editor/d2l-activity-evaluation-editor/d2l-activity-evaluation-settings-editor.js]: Good catch. I missed it. Thank you!

**Review by MingoLi — DISMISSED** [review]

**Review by jmurray-d2l — APPROVED** [review]

**Pull request merged**